### DISCUSSION TITLE
[Question]: Database not found

### DISCUSSION THREAD

**Discussion by a13782425** [danny-avila/LibreChat#1880] [Troubleshooting] [answered]
### What is your question?

MongoDb will be deleted after a period of operation

### More Details

[error-2024-02-24.log](https://github.com/danny-avila/LibreChat/files/14393053/error-2024-02-24.log)
environment: docker

### What is the main subject of your question?

User System/OAuth

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by a13782425** [comment]
any solutions there

**Comment by a13782425** [comment]
LibreChat is running but cannot see the database. It's normal when it starts up
![pic1](https://github.com/danny-avila/LibreChat/assets/11312875/301ad843-e049-4d30-a91c-b08330304186)

![pic2](https://github.com/danny-avila/LibreChat/assets/11312875/30fa9780-6a29-44db-9dee-67feabe27868)

**Reply by fuegovic** [reply, marked as answer]
@a13782425 
Looks like you exposed your database ports and it was hacked. 
You have a couple of possible solutions here:
- Not exposing the ports and use mongo-express instead to manage it: [manage_your_database](https://docs.librechat.ai/features/manage_your_database.html)
- Password protect your database: [mongodb-authentication](https://docs.librechat.ai/install/configuration/docker_override.html#mongodb-authentication)
- Use an online database: [online-mongodb](https://docs.librechat.ai/install/configuration/mongodb.html) (also see the override example if you choose this option)

**Reply by a13782425** [reply]
I'll try to turn off the database's external ports.

Thank you. The database port is closed and it works
